calculation.py

```diff
@@ -0,0 +1,7 @@
+
+def add_five(x):
+    return x + 5
```

Comment:
This looks good but could you add some documentation like a docstring to explain the function.